src/content/api/compilation-hooks.md

```diff
@@ -418,7 +418,7 @@ Triggered after chunk `id` optimization has finished.
 
 `SyncHook`
 
-Store module info to the records. This is triggered only if 
```

Comment:
By the way, I noticed this incomplete sentence while editing, but wasn't sure what should be added to it.

Reply:
can you raise an issue an issue about it, we will pick at up